web/plugs/current_user.ex

```diff
@@ -1,19 +1,18 @@
 defmodule CodeCorps.Plug.CurrentUser do
-  alias CodeCorps.GuardianSerializer
+  @moduledoc """
+  Puts authenticated Guardian user into conn.assigns[:current_user]
+  """
 
+  @spec init(Keyword.t) :: Keyword.t
   def init(opts), do: opts
 
+  @spec call(Plug.Conn.t, Keyword.t) :: Plug.Conn.t
   def call(conn, _opts) do
-    case Guardian.Plug.current_token(conn) do
+    case Guardian.Plug.current_resource(conn) do
+      user = %CodeCorps.User{} ->
+        Plug.Conn.assign(conn, :current_user, user)
       nil ->
         conn
-      current_token ->
```

Comment:
I see you removed this completely. Is it your experience that this case does not happen at all? Makes me wonder why we had it in the first place. @JoshSmith Any ideas?